includes/managers/class-fs-checkout-manager.php

```diff
@@ -153,7 +181,40 @@ public function get_query_params( Freemius $fs, $plugin_id, $plan_id, $licenses
 				( $fs->is_theme() && current_user_can( 'install_themes' ) )
 			);
 
-			return array_merge( $context_params, $_GET, array(
+            /**
```

Comment:
Remove this doc block please. See that it is already stale. Instead

1. See the `start.php` and see we have list of `Freemius filters collection`, just add the filter there.
2. Later we will update our docs https://freemius.com/help/documentation/wordpress-sdk/filters-actions-hooks/